notebooks/Model_Training_with_TensorArray.ipynb

```diff
@@ -0,0 +1,5826 @@
+{
```

Comment:
<p>"Standard arithetic and comparison ..." -&gt; "Standard <strong>arithmetic</strong> and comparison ..."</p><p> </p><p>"... elements of similar shaprs." -&gt; "...  elements of similar shapes."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>